spec/migrations/20201227173629_update_startup_shortcut_after_policy_profile_deexplorization_spec.rb

```diff
@@ -0,0 +1,47 @@
+require_migration
+
+describe UpdateStartupShortcutAfterPolicyProfileDeexplorization do
+  let(:user_stub) { migration_stub :User }
+
+  migration_context :up do
+    describe 'starting page update' do
+      it 'update user start page if miq_policy_set/explorer' do
+        user = user_stub.create!(:settings => {:display => {:startpage => 'miq_policy_set/explorer'}})
+
+        migrate
+        user.reload
+
+        expect(user.settings[:display][:startpage]).to eq('miq_policy_set/show_list')
+      end
+    end
+
+    it 'does not affect users without settings' do
+      user = user_stub.create!
+
+      migrate
+
+      expect(user_stub.find(user.id)).to eq(user)
```

Comment:
I'm not sure what this is testing? Should it check that a user that has a different start page set would still have the same page set after the migration?

Reply:
@gtanzillo It just tests if user did not ave any settings set at all remain the unchanged, but i can add another test to test the case you mentioned in your comment

Reply:
@gtanzillo please re-review